src/signals/incident-management/containers/Dashboard/components/BarChart/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface RawData {
+  total: number
+  results: []
```

Comment:
Arent they any array?

Reply:
hmm, true. I made it `undefined[]`, which is technically not true since there won't be any undefined item in the array, but is as close as it gets. Couldn't find a typescript type that declares an array that is always empty.